soaks/bin/analyze_experiment

```diff
@@ -127,16 +128,14 @@ p_value_violation = results['p-value'] < args.p_value
 erratic_violation = results.erratic == True
 
 changes = results[p_value_violation & drift_filter & ~declared_erratic].copy(deep=True)
-changes['Δ confidence'] = changes['p-value'].apply(common.confidence)
```

Comment:
I thought we still wanted the new condifence column instead of p-value?

Reply:
eh, ignore this. I read it wrong

Reply:
Ah, yeah, I moved it up so it appears in both tables.